### PR TITLE
change test script, add coverage script

### LINKED ISSUE #156: Update package: add script for code coverage assessment
- Modify test command, removing the coverage flag and setting it to dev mode.
- Add a coverage command that duplicates the existing test command.

### PR DESCRIPTION
**Pull request by ashfox21**
Added a 'coverage' script that runs tests with code coverage assessment.
Rewritten the 'test' script: it now excludes coverage and runs tests in development mode (dev).

### CODE REVIEW

**Comment by OliaHrytsak** [comment]
I think it's not necessary changes for project, because it is very short and now we already pass the half of way)

**Review by Wiret02 — APPROVED** [review]

**Review by holus-bolus — APPROVED** [review]

**Review by VictoriaRodik — APPROVED** [review]

**Review by StasRusnak1 — APPROVED** [review]

**Review by OliaHrytsak — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -68,7 +68,8 @@
   "scripts": {
     "start": "vite",
     "build": "vite build",
-    "test": "cross-env CI=true vitest --coverage",
+    "test": "cross-env CI=true vitest dev",
```
Comment (vlad-khrychov) [inline package.json]: You changed `test` script, but this script is used in `.husky/pre-push`. Maybe you should only add new script without changing of existing?

**Review by vlad-khrychov — CHANGES_REQUESTED** [review]

**Comment by vlad-khrychov** [comment]
> Added a 'coverage' script that runs tests with code coverage assessment. Rewritten the 'test' script: it now excludes coverage and runs tests in development mode (dev).

I think we need only a new script like `test-dev`

**Review by holus-bolus — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (ashfox21) [reply package.json]: Returned the original test command and added a new test-dev command to test in dev mode.

**Review by vlad-khrychov — APPROVED** [review]

**Pull request merged**